Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,13 +1,12 @@
 @echo off
 
-rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
-    set "_OLD_CODEPAGE=%%a"
-)
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it.
+for /f %%a in ('__VENV_PYTHON__ -s -S -E -c "from ctypes import windll; print(windll.kernel32.GetConsoleOutputCP());"') do (set "_OLD_CODEPAGE=%%a")
+for /f %%a in ('__VENV_PYTHON__ -s -S -E -c "from ctypes import windll; print(windll.kernel32.GetConsoleCP());"') do (set "_OLD_INPUT_CODEPAGE=%%a")
```

Comment:
We don't need to save, set, and restore the console input codepage. CMD decodes the script using the console output codepage, so that's the only one that needs to temporarily change to UTF-8.

Reply:
I have tried to be closer to the original behaviour of the scripts as chcp will still return the original codepage when just setting the ConsoleOutputCP:

>python.exe -s -S -E -c "import ctypes; ctypes.windll.kernel32.SetConsoleOutputCP(65001);"

>chcp
Aktive Codepage: 850.

But when:
`>python.exe -s -S -E -c "import ctypes; ctypes.windll.kernel32.SetConsoleCP(65001); ctypes.windll.kernel32.SetConsoleOutputCP(65001);"`

>chcp
Aktive Codepage: 65001.

Reply:
I explained previously that chcp.com returns the input codepage but sets both the input and output codepages. This means the original script potentially loses the initial output codepage. It's only an issue if they're different, which is a rare configuration. The point of using ctypes instead of chcp.com, other than working around the localization problem, is to focus on saving, setting, and restoring just the output codepage, which is all that's needed to make CMD correctly decode this UTF-8 batch script.

Reply:
Thanks for the extended explanation. I have removed the unneeded code in the new commit. I would like to squash all the commits together and add the useful explanation in the commit message. What do you think about the parameters I use to call python: -s -S -E? 

-E  Ignore all PYTHON* environment variables, e.g. PYTHONPATH and PYTHONHOME, that might be set.
-s   Don’t add the user site-packages directory to sys.path.
    See also    PEP 370 – Per user site-packages directory
-S  Disable the import of the module site and the site-dependent manipulations of sys.path that it entails. Also disable these manipulations if site is explicitly imported later (call site.main() if you want them to be triggered.)